apps/condo/domains/resident/schema/Resident.js

```diff
@@ -198,6 +227,30 @@ const Resident = new GQLListSchema('Resident', {
                 resolveInput: getUnitTypeFieldResolveInput(),
             },
         },
+        isVerifiedByManagingCompany: {
```

Comment:
Why not just `isVerified`? It's to complex field name. Also we already have fields with same logic

Reply:
It will not be as clear for API users as current name.

Most likely, soon information about not only the **managing company**, but also about **the service providers** will be included here. This naming emphasizes the link to the managing company, as I cannot guarantee that in some future we will not have to add similar fields for service providers.